test/e2e_flags.go

```diff
@@ -44,6 +44,7 @@ type EnvironmentFlags struct {
 }
 
 func initializeFlags() *EnvironmentFlags {
+	initTesting()
```

Comment:
Would wrap this with thunk be a bit better? https://godoc.org/github.com/m90/go-thunk

Reply:
I don't think capturing panic in that call makes much sense, given it's a testing file.